### ISSUE TITLE
Uncaught Error: Call to undefined function SendGrid\curl_init()

### ISSUE THREAD

**Issue by internationalhouseofdonald** [sendgrid/sendgrid-php#811]
#### Issue Summary

A summary of the issue and the environment in which it occurs. If suitable, include the steps required to reproduce the bug. Please feel free to include screenshots, screencasts, code examples.

------

I have the [basic example](https://github.com/sendgrid/sendgrid-php#hello-email) set up in a local development environment (not encrypted) and also a remote testing environment (encrypted). At first, when I was working in my local development environment, a call to `$sendgrid->send($email);` would return an `{}` empty object, so thinking that it could be the insecure connection, I moved the code over to a remote testing environment that is running a secure connection (HTTPS). At first, in the remote testing environment, I was getting a 500 Internal Server error so I followed [these instructions](https://stackoverflow.com/questions/17693391/500-internal-server-error-for-php-file-not-for-html) to output the error. Here's what the error said:

`Fatal error: Uncaught Error: Call to undefined function SendGrid\curl_init() in /<path>/<to>/sendgrid-php/vendor/sendgrid/php-http-client/lib/Client.php:459 Stack trace: #0 /<path>/<to>/sendgrid-php/vendor/sendgrid/php-http-client/lib/Client.php(591): SendGrid\Client->makeRequest('post', 'https://api.sen...', Object(SendGrid\Mail\Mail), NULL, false) #1 /<path>/<to>/sendgrid-php/lib/SendGrid.php(75): SendGrid\Client->__call('post', Array) #2 /<path>/<to>/test.php(24): SendGrid->send(Object(SendGrid\Mail\Mail)) #3 {main} thrown in /<path>/<to>/sendgrid-php/vendor/sendgrid/php-http-client/lib/Client.php on line 459`

I tried enabling curl within php.ini like in Issue #8 but to no avail.

#### Steps to Reproduce

------

1. Download [the SendGrid PHP API](https://github.com/sendgrid/sendgrid-php/releases/download/v7.2.1/sendgrid-php.zip) as a .zip file and extract all. `scp` it over to a secure server.
2. Set up [the Hello World example](https://github.com/sendgrid/sendgrid-php/releases/download/v7.2.1/sendgrid-php.zip) with the SendGrid PHP API over a secure connection.
3. Run the script by making a simple GET request to the page.

> Any other information you want to share that is relevant to the issue being reported. Especially, why do you consider this to be a bug? What do you expect to happen instead?

I wouldn't call it a bug; I'm probably just implementing the code wrong. It could be the fact that I'm not using Composer. I would expect `curl_init()` to at least return some kind of error if it isn't working. I understand that's a lot to ask but I've been working on this for a little over 24 hours now.

#### Technical details:

* sendgrid-php Version: master (latest commit: [commit number])
* PHP Version: 7.0

**Comment by internationalhouseofdonald** [comment]
I'm slowly figuring out that this is a problem with the fact that I'm calling the PHP function `curl_init()` and cURL isn't enabled on my system.

**Comment by internationalhouseofdonald** [comment]
So, this was all because I was having a little trouble installing `php7.0-curl` on my DO droplet. Here are the steps that I took to fix the issue:

1. sudo apt-add-repository ppa:ondrej/php
2. sudo apt-get update
3. sudo apt-get install php7.0 (questionable, but it worked.)
4. sudo apt-get install php7.0-curl
5. sudo service apache2 restart

I was also having an issue with my API key. I was wrapping my `SENDGRID_API_KEY` environment variable in single quotes and it was causing this error: 

> The provided authorization grant is invalid, expired, or revoked (401)

TLDR: If `curl_init()` isn't working, then you have to install cURL on your system first.

**Comment by internationalhouseofdonald** [comment]
Further down the rabbit hole:

On the `7.0.33-2+ubuntu18.04.1+deb.sury.org+2+will+reach+end+of+life+in+april+2019` issue, I had PHP 7.2 already installed, so all I did was run these commands to get that warning message to go away:

1. sudo a2dismod php7.0
2. sudo a2enmod php7.2
3. systemctl restart apache2
